uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/OrchestratorZoneControllerIntegrationTests.java

```diff
@@ -308,27 +330,28 @@ private void testAdminClientSecretAsSpaceAndEmpty(OrchestratorZoneRequest orches
             String.class);
         assertEquals(getResponse.getStatusCode(), HttpStatus.BAD_REQUEST);
         assertEquals(APPLICATION_JSON_UTF8, getResponse.getHeaders().getContentType());
-        String expected  =  JsonUtils.writeValueAsString(
-            new OrchestratorErrorResponse("The adminClientSecret field cannot contain" +
-                                          " spaces or cannot be blank."));
-        assertEquals(expected, getResponse.getBody());
+        assertTrue(getResponse.getBody().contains("parameters.adminClientSecret " +
+                                                  "must not be empty and must not have empty spaces"));
     }
 
     private void testWithSpaceOrSpecialCharFail(OrchestratorZoneRequest orchestratorZoneRequest) {
         ResponseEntity<String> getResponse = client.exchange(
             serverRunning.getUrl("/orchestrator/zones"), HttpMethod.POST, new HttpEntity<>(orchestratorZoneRequest),
             String.class);
         assertEquals(getResponse.getStatusCode(), HttpStatus.BAD_REQUEST);
-        assertTrue(getResponse.getBody().contains("Special characters are not allowed in the subdomain " +
-                                                  "name except hyphen which can be specified in the middle."));
+        assertEquals(APPLICATION_JSON_UTF8, getResponse.getHeaders().getContentType());
+        assertTrue(getResponse.getBody().contains("parameters.subdomain " +
+                                                  "special characters are not allowed in the subdomain name " +
+                                                  "except hyphen which can be specified in the middle"));
```

Comment:
@babamishra9 Please confirm this error message with pushkaraj for its format.

Reply:
As discussed, error messages are updated.